org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -7053,9 +7053,12 @@ class FileOperationPattern {
 	/**
 	 * Whether to match files or folders with this pattern.
 	 *
+	 * The match kind could be <code>FileOperationPatternKind.File</code>,
```

Comment:
Use `{@link FileOperationPatternKind}` to reference the source of values.

Reply:
Nice suggestion!